### DISCUSSION TITLE
cannot connect to DB e and to the EDGE !

### DISCUSSION THREAD

**Discussion by RankingRoadIt** [sqlitecloud/discussions#57] [Help] [answered]
I have no idea when it started, but apparently the database cannot be open and the relative edge functions are lost too.

[SQLite Cloud Bug Report.zip](https://github.com/user-attachments/files/18180902/SQLite.Cloud.Bug.Report.zip)

{
  "info": {
    "error": {
      "status": "500",
      "title": "SQLiteCloudError: Unable to open database edge.sqlitecloud (N/A).",
      "detail": "GET /v2/functions an error occured"
    }
  },
  "status": 500
}

**Comment by Gioee** [comment]
Hi @RankingRoadIt,
Would you like to share your project id without api key? So we can have a look at it.

**Comment by RankingRoadIt** [comment]
Of course the node is:  ndylkccriz.sqlite.cloud there are 3 databases and the disk is reported at 100% ?

**Comment by danielebriggi** [comment, marked as answer]
We had an issue on the Storage mounting. Please check that your service is working as expected

**Reply by RankingRoadIt** [reply]
nice! thanks !

is it something that i've done or it was an external issue ?

**Reply by danielebriggi** [reply]
it's was an issue on the infrastructure layer, beyond your control
